tests/args_parser.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright (c) 2020 Kamal Heib <kamalheib1@gmail.com>, Inc. All rights reserved.  See COPYING file
```

Comment:
You have "Inc." in the copyright line, did you do in on purpose?

Reply:
> You have "Inc." in the copyright line, did you do in on purpose?

Maybe in the future :-). I'll fix it in v3.